.github/workflows/release.yml

```
@@ -882,6 +885,63 @@ jobs:
                 pnpm install
                 pnpm moonwall test zombie_${{ matrix.chains.chain }}_upgrade
 
+    zombienet-tests-tanssi-relay:
+      runs-on: self-hosted
+      needs: ["set-tags", "build"]
+      steps:
+          - name: Checkout
+            uses: actions/checkout@v4
+            with:
+                ref: ${{ needs.set-tags.outputs.git_ref }}
+
+          - name: Pnpm
+            uses: pnpm/action-setup@v3.0.0
+            with:
+                version: 8
+
+          - name: Setup Node
+            uses: actions/setup-node@v4
+            with:
+                node-version: 20.x
+                cache: "pnpm"
+
+          - name: "Download binaries"
+            uses: actions/download-artifact@v4
+            with:
+                name: binaries
+                path: target/release
+
+          - name: "Run zombie test"
+            run: |
+                chmod uog+x target/release/tanssi-node
+                chmod uog+x target/release/tanssi-relay
+                chmod uog+x target/release/tanssi-relay-execute-worker
+                chmod uog+x target/release/tanssi-relay-prepare-worker
+                chmod uog+x target/release/container-chain-simple-node
+
+                cd test
+                pnpm install
+
+                ## Run tests
+
+                pnpm moonwall test zombie_tanssi_relay
+
+          - name: "Gather zombie logs"
+            if: failure()
+            run: |
+                ls -ltr /tmp
+                latest_zombie_dir=$(find /tmp -type d -iname "*zombie*" -printf '%T@ %p\n' | sort -n | tail -1 | cut -f2- -d" ")
+                logs_dir="logs"
+                mkdir -p "$logs_dir"
+                find "$latest_zombie_dir" -type f -name "*.log" -exec cp {} "$logs_dir" \;
+
+          - name: "Upload zombie logs"
+            if: failure()
+            uses: actions/upload-artifact@v4
+            with:
+                name: logs
```

Use a different name here to avoid conflicts with the other zombienet suites

```suggestion
                name: logs-tanssi-relay
```